### ISSUE TITLE
Uncaught TypeError: Cannot read property 'path' of undefined

### ISSUE THREAD

**Issue by marr975** [tomkadwill/atom-cucumber-autocomplete#18]
[Enter steps to reproduce below:]
1. Open Atom.
2. Select 'Gherkin' from the Language tray item.
3. Create a scenario within the feature.
4. Type either 'Given', 'When' or 'Then' in the scenario.

An error occurs when the autocomplete message appears.

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [cucumber-autocomplete](https://github.com/tomkadwill/atom-cucumber-autocomplete) package, v0.6.3
### Stack Trace

Uncaught TypeError: Cannot read property 'path' of undefined

```
At /Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:62

TypeError: Cannot read property 'path' of undefined
    at Object.rootDirectory (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:62:43)
    at Object.scanFeaturesDir (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:46:31)
    at Object.getCompletions (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:31:19)
    at Object.getSuggestions (/Users/markrodgers/.atom/packages/cucumber-autocomplete/lib/provider.js:17:17)
    at /Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:310:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:289:17)
    at AutocompleteManager.getSuggestionsFromProviders (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (/Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:276:19)
    at /Users/markrodgers/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61
```
### Commands

```
     -3:20 core:paste (atom-text-editor.editor.is-focused)
     -3:17.4.0 grammar-selector:show (atom-text-editor.editor)
  8x -3:01.9.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:02.8.0 editor:newline (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "language-rubymotion"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
autocomplete-motion, v2.1.0
cucumber, v0.5.0
cucumber-autocomplete, v0.6.3
language-r, v0.4.1
language-rspec, v0.5.0
language-scala, v1.1.6

# Dev
No dev packages
```

**Comment by tomkadwill** [comment]
Hi @marr975, thanks for reporting this. Sorry for the delay getting back to you!

The plugin doesn't work when you haven't saved a project to the filesystem. In this case, the plugin definitely shouldn't throw an exception. I'll update the package so that no error is thrown in this case and the plugin becomes 'inactive'

**Comment by tomkadwill** [comment]
Hi @marr975, I've published a fix for this in v0.7.2. Please re-open this ticket if v0.7.2 does not fix the issue.
